docs/docs/getting-started/cluster-setup.md

```diff
@@ -2,17 +2,17 @@
 
 ## Prerequisites
 
-Install the latest version of the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/).
+* Install a recent version of the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/). Note that `apt install` will likely install an outdated version.
```

Comment:
We know the exact version that's needed (2.44.1) from the [Azure docs](https://learn.microsoft.com/en-us/azure/aks/deploy-confidential-containers-default-policy#prerequisites). I'd also replace *`apt install`* with *version installed by your package manager*.